typescript/sdk/src/consts/environments/mainnet.json

```diff
@@ -212,5 +212,22 @@
     "protocolFee": "0xd83A4F747fE80Ed98839e05079B1B7Fe037b1638",
     "validatorAnnounce": "0x2fa5F5C96419C222cDbCeC797D696e6cE428A7A9",
     "routingIsmFactory": "0x8358D8291e3bEDb04804975eEa0fe9fe0fAfB147"
+  },
+  "inevm": {
+    "merkleRootMultisigIsmFactory": "0x2C1FAbEcd7bFBdEBF27CcdB67baADB38b6Df90fC",
+    "messageIdMultisigIsmFactory": "0x8b83fefd896fAa52057798f6426E9f0B080FCCcE",
+    "aggregationIsmFactory": "0x8F7454AC98228f3504Bb91eA3D8Adafe6406110A",
+    "aggregationHookFactory": "0xEb9FcFDC9EfDC17c1EC5E1dc085B98485da213D6",
+    "routingIsmFactory": "0x1052eF3419f26Bec74Ed7CEf4a4FA6812Bc09908",
+    "domainRoutingIsm": "0xBD70Ea9D599a0FC8158B026797177773C3445730",
+    "proxyAdmin": "0x0761b0827849abbf7b0cC09CE14e1C93D87f5004",
+    "storageGasOracle": "0x6119E37Bd66406A1Db74920aC79C15fB8411Ba76",
+    "interchainGasPaymaster": "0x19dc38aeae620380430C200a6E990D5Af5480117",
+    "merkleTreeHook": "0x0972954923a1e2b2aAb04Fa0c4a0797e5989Cd65",
+    "aggregationHook": "0xe0dDb5dE7D52918237cC1Ae131F29dcAbcb0F62B",
+    "protocolFee": "0x0D63128D887159d63De29497dfa45AFc7C699AE4",
+    "mailbox": "0x2f2aFaE1139Ce54feFC03593FeE8AB2aDF4a85A7",
+    "validatorAnnounce": "0x15ab173bDB6832f9b64276bA128659b0eD77730B",
+    "interchainSecurityModule": "0xBD70Ea9D599a0FC8158B026797177773C3445730"
```

Comment:
testRecipient & testTokenRecipient?
Looks like they're missing for manta above too